src/logic/operate.js

```diff
@@ -5,16 +5,16 @@ const operate = (numberOne, numberTwo, operation) => {
   const numTwo = Big(numberTwo);
   switch (operation) {
     case '+':
-      return numOne.plus(numTwo);
+      return numOne.plus(numTwo).toString();
     case '-':
-      return numOne.minus(numTwo);
+      return numOne.minus(numTwo).toString();
     case '÷':
       if (numTwo === 0) return 0;
```

Comment:
Hi, this is breaking the app `===` checks for type, don't forget button names are strings.

Reply:
Done! 👍